lib/mjml.rb

```diff
@@ -5,7 +5,7 @@
 
 require 'mjml/handler'
 require 'mjml/parser'
-
+require 'mjml/mrml_parser' if defined?(MRML)
```

Comment:
This doesn't work for me in my Rails project even after installing `gem 'mrml'` and setting `config.use_mrml = true` in my initialiser.

When I comment out `if defined?(MRML)` it renders using MRML fine (very quickly!). Am I missing something else?

Here's the error I see:

```bash
NameError - uninitialized constant Mjml::MrmlParser:
  app/views/issues/email.mjml:1

```

Reply:
Looks like the order of gems in the `Gemfile` is important. I had `gem "mrml"` before `gem "mjml-rails"`, but if `mjml-rails` is first it fails. Maybe I can get rid of `if defined?(MRML)` completely, people will simply see an error if they forget to add `mrml` to their Gemfile.

Reply:
Okay I pushed a fix, can you check now?